### ISSUE TITLE
Cannot read endpointFile from environment Variable

### ISSUE THREAD

**Issue by ftrossbach** [localstack/serverless-localstack#70]
I used to pass the location of my endpoint file like this:

```
custom:
  localstack:
    stages:
      - local
    endpointFile: ${env:ENDPOINT_FILE, './tests/dummy.json'}
````

For some reason, that ceased to work starting with version 0.4.21 (works up to 0.4.20) and throws the following error:
`ReferenceError: Endpoint: "${env:ENDPOINT_FILE, './tests/dummy.json'}" is invalid: Error: ENOENT: no such file or directory, open '${env:ENDPOINT_FILE, './tests/dummy.json'}'`

This suggests to me that the variable is no longer evaluated and serverless-localstack tries to open a file called ${env...}, but looking at the source code I don't see any relevant changes that ought to trigger such a change in behaviour (but then again I'm not very experienced in Node.js). 

Any help on this is highly appreciated.

**Comment by zhenwenc** [comment]
I can't use environment variable to configure the localstack `host` as well... sigh...

**Comment by whummer** [comment]
Thanks for reporting @ftrossbach @zhenwenc . This should be fixed in 06f48d9e3125fffe626592e38c2b821fc0ba0f77 - can you please give it another try with the latest version `serverless-localstack@0.4.27`?

Also, please note that the service-specific ports have been deprecated in the latest version of LocalStack - the new approach is to use a central edge service port (4566 by default) as the central entry point.

Hope that helps - please let us know if the problem persists. Thanks
